cli/command/service/trust.go

```diff
@@ -15,10 +15,9 @@ import (
 	"golang.org/x/net/context"
 )
 
-func resolveServiceImageDigest(dockerCli command.Cli, service *swarm.ServiceSpec) error {
+func resolveServiceImageDigestContentTrust(dockerCli command.Cli, service *swarm.ServiceSpec) error {
 	if !command.IsTrusted() {
-		// Digests are resolved by the daemon when not using content
-		// trust.
+		// Digests are resolved in the client when not using content trust.
```

Comment:
This comment is strange. By default, the digest is resolved on the client either way.